### PR TITLE
add googlemaps package

### PR DESCRIPTION
**Pull request by david-gang**
add googlemaps package to conda-forge.

### CODE REVIEW

**Comment by david-gang** [comment]
It fails because Be sure to add all dependencies in the meta.yaml  url=https://pypi.python.org/simple/requests/
What's weird is that this is a dependency of the package and not the package itself. And also why doesn't it point to the conda package

**Comment by david-gang** [comment]
I tried to build it also manually. I installed the requests library and still things are not working. This is weird

**Comment by david-gang** [comment]
Missed the setup-requires :-) Now fixed.
https://groups.google.com/a/continuum.io/forum/?utm_medium=email&utm_source=footer#!topic/conda/9edHVCBK0Ww

File: recipes/googlemaps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - requests >=2.11.1,<3.0
+
+test:
+  imports:
+    - googlemaps
+
+about:
+  home: http://github.com/simplejson/simplejson
+  license: Apache 2
```
Comment (synapticarbors) [inline recipes/googlemaps/meta.yaml]: `Apache-2.0`

https://opensource.org/licenses/alphabetical

File: recipes/googlemaps/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  license_family: Apache
```
Comment (synapticarbors) [inline recipes/googlemaps/meta.yaml]: Please manually package the license and add an issue upstream:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

**Pull request merged**